js/widgets/navbar.backcompat.js

```diff
@@ -0,0 +1,67 @@
+/*!
+ * jQuery Mobile Navbar @VERSION
+ * http://jquerymobile.com
+ *
+ * Copyright jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ */
+
+//>>label: Navbar Backcompat
+//>>group: Widgets
+//>>description: Packcompat Formats groups of links as horizontal navigation bars.
+//>>docs: http://api.jquerymobile.com/navbar/
+//>>demos: http://demos.jquerymobile.com/@VERSION/navbar/
+//>>css.structure: ../css/structure/jquery.mobile.navbar.css
+//>>css.theme: ../css/themes/default/jquery.mobile.theme.css
+( function( factory ) {
+	if ( typeof define === "function" && define.amd ) {
+
+		// AMD. Register as an anonymous module.
+		define( [
+			"jquery",
+			"./navbar",
+			"./widget.backcompat" ], factory );
+	} else {
+
+		// Browser globals
+		factory( jQuery );
+	}
+} )( function( $ ) {
+
+if ( $.mobileBackcompat !== false ) {
+	return $.widget( "mobile.navbar", $.mobile.navbar, {
+		_create: function() {
+			var self = this;
```

Comment:
Don't use self this break in some contexts self has special meaning `window.self === window` instead use `var that = this;`